ithaka-chatpopup/README.md

```
@@ -0,0 +1,111 @@
+npm run dev
+
+# Construir
+npm run build
+
+# Limpiar
+npm run clean
+```
+
+## 📄 Licencia
+
+MIT 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)